en/flight_controller/kakuteh7mini.md

```diff
@@ -0,0 +1,143 @@
+# Holybro Kakute H7 mini
+
+:::warning
+PX4 does not manufacture this (or any) autopilot.
+Contact the [manufacturer](https://shop.holybro.com/) for hardware support or compliance issues.
+:::
+
+The [Holybro Kakute H7 mini](http://www.holybro.com/product/kakute-h7-mini/) flight controller is intended for lightweight frame builds (such as racers, etc.).
+
+ This flight controller is full of features including integrated Bluetooth, HD camera plug, dual plug-and-play 4in1 ESC ports, 9V VTX ON/OFF Pit Switch, barometer, OSD, 6x UARTs, 128MB Flash for logging (not supported with PX4 yet), 5V BEC, and bigger soldering pad with easy layout and much more.
```

Comment:
@vincentpoont2 In your comment in https://github.com/PX4/PX4-user_guide/pull/2247#discussion_r1099020115 , I think you are saying that bluetooth and the Pit switch are no on the mini - right?

```suggestion
 This flight controller has many useful features, including integrated HD camera plug, dual plug-and-play 4in1 ESC ports, barometer, OSD, 6x UARTs, 128MB Flash for logging (not supported with PX4 yet), 5V BEC, and bigger soldering pad with easy layout and much more.
```

Reply:
@vincentpoont2 This is waiting on final check for you here!

Reply:
@hamishwillee Sorry, it has the VTX pit switch, but no bluetooth. I have made the edit.